src/main/io/displayport_max7456.c

```diff
@@ -154,7 +156,7 @@ static const displayPortVTable_t max7456VTable = {
 displayPort_t *max7456DisplayPortInit(const vcdProfile_t *vcdProfile)
 {
     displayInit(&max7456DisplayPort, &max7456VTable);
-    max7456Init(vcdProfile);
+    max7456Init(vcdProfile, systemConfig()->cpu_overclock);
```

Comment:
This breaks on SPRACINGF3OSD, thanks to `cpu_overclock` being removed from the parameter group for this target.

Reply:
max7456Init can query actual bus speed to determine if overclock is possible. It should be even more robust and future-safe